erpnext/templates/form_grid/item_grid.html

```diff
@@ -21,6 +21,13 @@
 					{%= doc.sales_order || doc.against_sales_order %}
 				</span></p>
 			{% } %}
+			{% if(doc.purchase_order || doc.against_purchase_order) { %}
+				<p><span class="label label-default"
+					title="{%= __("Purchase Order") %}">
+					<i class="icon-file"></i>
+					{%= doc.purchase_order || doc.against_purchase_order %}
+				</span></p>
+			{% } %}
 			{% include "templates/form_grid/includes/visible_cols.html" %}
```

Comment:
We can remove the icon and label. As per the new styling we can just leave it a muted text
```
{% if(doc.purchase_order || doc.against_purchase_order) { %}
   <p class="text-muted">{%= doc.purchase_order || doc.against_purchase_order %}</p>
{% } %}
```